src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fs">
+        <legend>Personal information</legend>
+        <div class="formfield">
+          <label for="surname">
+                  Surname: <input type="text" required
+name="surname" id="surname">
+          </label>
+
+        </div>
+        <div class="formfield">
+          <label for="name">
+               Name: <input type="text" minlength="3"
+             maxlength="20" autocomplete="off"
+name="name" id="name">
+          </label>
+
+        </div>
+        <div class="formfield">
+          <label for="old">
+            How old are You? <input type="number" value="12"
+             name="old" id="old"
+             min="1" max="100"
+           autocomplete="off">
+          </label>
+
+        </div>
+        <div class="formfield">
+          <label for="date">
+            Full date of birth: <input type="date" autocomplete="off"
+           name="date" id="date">
```

Comment:
let's write more meaningful names for inputs, `dateOfBithDay` will be better. Fix in all cases